applications/openshift/api-server/api_server_kubelet_https_key/rule.yml

```diff
@@ -0,0 +1,54 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Enable kubelet HTTPS connections to the API Server'
+
+description: |-
+    By default, the OpenShift API Server runs in HTTPS mode. HTTPS should be
+    used for connections between the API Server and Kubelets.
+    Edit the <tt>openshift-kube-apiserver</tt> configmap on the master node(s)
+    and remove the <tt>kubelet-https</tt> parameter if it is set to <tt>false</tt>:
+    <pre>
+    "apiServerArguments":{
+      ...
+      "kubelet-https":[
+        "false"
+      ],
+      ...
+    </pre>
```

Comment:
you might want to change the description to say that you're checking the key

Reply:
Thanks, updated